packages/devtools_server/lib/src/server.dart

```diff
@@ -514,47 +510,47 @@ ArgParser _createArgsParser(bool verbose) {
       help:
           'Launches DevTools in a browser immediately at start.\n(defaults to on unless in --machine mode)',
     )
-// TODO: Remove this - prefer that clients use the rest arg.
-    ..addOption(
-      argVmUri,
-      defaultsTo: '',
-      help: 'VM Service protocol URI.',
-      hide: true,
-    )
     ..addFlag(
       argMachine,
       negatable: false,
       help: 'Sets output format to JSON for consumption in tools.',
-    )
+    );
+
+  parser.addSeparator('Memory profiling options:');
+
+  parser
     ..addOption(
       argProfileMemory,
       valueHelp: 'file',
       defaultsTo: 'memory_samples.json',
       help:
           'Start devtools headlessly and write memory profiling samples to the '
           'indicated file.',
-    )
-// TODO: Remove this after a release or two.
-    ..addOption(
-      argProfileMemoryOld,
-      defaultsTo: 'memory_samples.json',
-      hide: true,
-    )
+    );
+
+  if (verbose) {
+    parser.addSeparator('App size options;');
+  }
+
+  parser
     ..addOption(
       argAppSizeBase,
```

Comment:
Longer term, we way want to deprecate `--appSizeBase` and `--appSizeTest` in favor of naming the options something like `--app-size-base` and `--app-size-test`.

Reply:
Would you mind filing an issue for this and linking it in a TODO here?

Reply:
done! https://github.com/flutter/devtools/issues/3146